ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -584,271 +463,41 @@ const PanelMain = fp.flow(
-                    })
-                  }
-                  disabled={disableControls}
-                  disableDetachableReason={disableDetachableReason}
-                  existingDisk={gcePersistentDisk}
-                  computeType={analysisConfig.computeType}
-                />
-                {runtimeExists && updateMessaging.warn && (
-                  <WarningMessage iconSize={30} iconPosition={'center'}>
-                    <div>{updateMessaging.warn}</div>
-                  </WarningMessage>
-                )}
-                {getErrorMessageContent().length > 0 && (
-                  <ErrorMessage
-                    iconSize={16}
-                    iconPosition={'top'}
-                    data-test-id={'runtime-error-messages'}
-                  >
-                    {getErrorMessageContent()}
-                  </ErrorMessage>
-                )}
-                {getWarningMessageContent().length > 0 && (
-                  <WarningMessage
-                    iconSize={16}
-                    iconPosition={'top'}
-                    data-test-id={'runtime-warning-messages'}
-                  >
-                    {getWarningMessageContent()}
-                  </WarningMessage>
-                )}
-                {unattachedPdExists ? (
-                  <FlexRow
-                    style={{
-                      justifyContent: 'space-between',
-                      marginTop: '1.125rem',
-                    }}
-                  >
-                    <DeletePersistentDiskButton
-                      onClick={() =>
-                        setPanelContent(PanelContent.DeleteUnattachedPd)
-                      }
-                    />
-                    {unattachedDiskNeedsRecreate
-                      ? renderNextWithDiskDeleteButton()
-                      : renderCreateButton()}
-                  </FlexRow>
-                ) : (
-                  <FlexRow
-                    style={{
-                      justifyContent: 'space-between',
-                      marginTop: '1.125rem',
-                    }}
-                  >
-                    <LinkButton
-                      style={{
-                        ...styles.deleteLink,
-                        ...(disableControls || !runtimeExists
-                          ? { color: colorWithWhiteness(colors.dark, 0.4) }
-                          : {}),
-                      }}
-                      aria-label='Delete Environment'
-                      disabled={disableControls || !runtimeExists}
-                      onClick={() =>
-                        setPanelContent(PanelContent.DeleteRuntime)
-                      }
-                    >
-                      Delete Environment
-                    </LinkButton>
-                    {cond(
-                      [runtimeExists, () => renderNextUpdateButton()],
-                      [
-                        unattachedDiskNeedsRecreate,
```

Comment:
I removed this condition because it's unreachable.

It requires a logical contradiction: !unattachedPdExists && unattachedDiskNeedsRecreate